sycl/source/kernel_bundle.cpp

```diff
@@ -372,7 +374,9 @@ bool is_source_kernel_bundle_supported(backend BE, source_language Language) {
   // and it's support is limited to the opencl and level_zero backends.
   bool BE_Acceptable = (BE == sycl::backend::ext_oneapi_level_zero) ||
                        (BE == sycl::backend::opencl);
-  if ((Language == source_language::opencl) && BE_Acceptable) {
+  if ((Language == source_language::opencl ||
+       Language == source_language::spirv) &&
+      BE_Acceptable) {
     return detail::OpenCLC_Compilation_Available();
```

Comment:
Why does SPIR-V language support need "OpenCLC_Compiation_Available()"  ? I think this function should probably be re-written with the addition of SPIRV support